tests/framework/ROM/TimeSeries/ARMA/basic.xml

```diff
@@ -76,6 +76,9 @@
       <outTruncation domain="positive">Speed</outTruncation>
       <Fourier>604800, 302400, 86400, 43200</Fourier>
       <P>2</P>
+	  <!--<Fourier>604800, 302400, 86400, 43200, 720</Fourier>-->
```

Comment:
Did you mean to change this file?

Reply:
I think Tanaya changed that for experimentation and accidentally added it to tracked files. I'll change it back.